content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -159,14 +159,19 @@ The following methods exist for installing kubectl on Linux:
    ```shell
    curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-archive-keyring.gpg
    ```
+   On Debian 12 (bookworm) import the Google Cloud signing key with:
 
-3. Add the Kubernetes `apt` repository:
+   ```shell
+   curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/kubernetes-archive-keyring.gpg
```

Comment:
This change goes against Debian [security advice](https://wiki.debian.org/DebianRepository/UseThirdParty#OpenPGP_certificate_distribution) for using third party repos.

The advice in our docs:
```markdown
In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
You can create this directory if you need to, making it world-readable but writeable only by admins.
```
might be wrong if Debian bookworm doesn't have `/etc/apt/keyrings`. We would be willing to fix that instead

/hold
We should **not** make the proposed change.